packages/suite/src/components/wallet/UtxoSelection.tsx

```diff
@@ -0,0 +1,202 @@
+    margin-top: 2px;
+    ${({ isChecked, $isGrey, theme }) =>
+        isChecked &&
+        $isGrey &&
+        css`
+            > div:first-child {
+                background: ${theme.TYPE_LIGHTER_GREY};
+            }
+
+            :not(:hover) > div:first-child {
+                border: 2px solid ${theme.TYPE_LIGHTER_GREY};
+            }
+        `};
+`;
+
+const Address = styled.div`
+    overflow: hidden;
+    text-overflow: ellipsis;
+`;
+
+const ChangeAddress = styled.span`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    white-space: nowrap;
+`;
+
+const StyledCryptoAmount = styled(FormattedCryptoAmount)`
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    margin-left: auto;
+    padding-left: 4px;
+    white-space: nowrap;
+`;
+
+const TransactionDetail = styled.button`
+    background: none;
+    border: none;
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    cursor: pointer;
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    padding: 0;
+    text-decoration: underline;
+`;
+
+const IconWrapper = styled.div`
+    margin-right: 8px;
+`;
+
+const StyledFiatValue = styled(FiatValue)`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    margin-left: auto;
+    padding-left: 4px;
+`;
+
+interface Props {
+    isChecked: boolean;
+    transaction?: WalletAccountTransaction;
+    utxo: AccountUtxo;
+}
+
+export const UtxoSelection = ({ isChecked, transaction, utxo }: Props) => {
+    const { openModal } = useActions({
+        openModal: modalActions.openModal,
+    });
+
+    const { account, network, selectedUtxos, toggleUtxoSelection } = useSendFormContext();
+    const theme = useTheme();
+
+    const isChangeAddress = utxo.path.split('/')[4] === '1';
+    const amountInBtc = (Number(utxo.amount) / 10 ** network.decimals).toString();
+    const outputLabel = account.metadata.outputLabels?.[utxo.txid]?.[utxo.vout];
+
+    const handleCheckbox = () => toggleUtxoSelection(utxo);
+    const showTransactionDetail: React.MouseEventHandler = e => {
+        e.stopPropagation(); // do not trigger the checkbox
+        if (transaction) {
+            openModal({ type: 'transaction-detail', tx: transaction });
+        }
+    };
+
+    return (
+        <Wrapper onClick={handleCheckbox}>
+            <StyledCheckbox
+                $isGrey={!selectedUtxos.length}
+                isChecked={isChecked}
+                onClick={handleCheckbox}
+            />
+            <Body>
+                <Row>
+                    <Address>{utxo.address}</Address>
+                    {isChangeAddress && (
+                        <>
+                            <Dot />
+                            <ChangeAddress>
+                                <Translation id="TR_CHANGE_ADDRESS" />
```

Comment:
@dahaca Where would be the tooltip? I can't imagine it from these few lines..

Reply:
Yes, i think the tooltip would be nice..

Reply:
> it may sound like call to action

I like to use a dash to emphasize that it's a single term, so "change-address" or "change-output", but I don't think it widely used.

Reply:
I would stick to `change address` or only `change`

With reference to Bitcoin Design
https://bitcoin.design/guide/glossary/address/#change-address

Reply:
@b-irving please decide 🙂

Reply:
let's solve it as a followup